<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWidget</class>
 <widget class="QWidget" name="MainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1168</width>
    <height>625</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWidget</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../sources/resource.qrc">
    <normaloff>:/icons/storage.ico</normaloff>:/icons/storage.ico</iconset>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>1131</width>
     <height>451</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="addGoodsBtn">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>70</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>新增商品</string>
   </property>
   <property name="icon">
    <iconset resource="../sources/resource.qrc">
     <normaloff>:/icons/add.ico</normaloff>:/icons/add.ico</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="removeGoodsBtn">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>70</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>删除商品</string>
   </property>
   <property name="icon">
    <iconset resource="../sources/resource.qrc">
     <normaloff>:/icons/delete.ico</normaloff>:/icons/delete.ico</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="uploadWareHouseBtn">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>70</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>商品入库</string>
   </property>
   <property name="icon">
    <iconset resource="../sources/resource.qrc">
     <normaloff>:/icons/upload.ico</normaloff>:/icons/upload.ico</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="exportWareHouseBtn">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>70</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>商品出库</string>
   </property>
   <property name="icon">
    <iconset resource="../sources/resource.qrc">
     <normaloff>:/icons/download.ico</normaloff>:/icons/download.ico</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="exportDataBtn">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>70</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>导出数据</string>
   </property>
   <property name="icon">
    <iconset resource="../sources/resource.qrc">
     <normaloff>:/icons/export.ico</normaloff>:/icons/export.ico</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="summeryDataBtn">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>70</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>数据汇总</string>
   </property>
   <property name="icon">
    <iconset resource="../sources/resource.qrc">
     <normaloff>:/icons/summary.ico</normaloff>:/icons/summary.ico</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="searchDataBtn">
   <property name="geometry">
    <rect>
     <x>1060</x>
     <y>20</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>数据查询</string>
   </property>
   <property name="icon">
    <iconset resource="../sources/resource.qrc">
     <normaloff>:/icons/search.ico</normaloff>:/icons/search.ico</iconset>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>20</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>输入商品编号：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="searchEdit">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>20</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>590</y>
     <width>421</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <italic>false</italic>
     <bold>true</bold>
     <underline>false</underline>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string>提示：双击表中数据即选中商品编号，后续操作就是针对该商品的操作</string>
   </property>
  </widget>
  <widget class="QPushButton" name="showAllBtn">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>70</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>显示数据</string>
   </property>
   <property name="icon">
    <iconset resource="../sources/resource.qrc">
     <normaloff>:/icons/show.ico</normaloff>:/icons/show.ico</iconset>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../sources/resource.qrc"/>
 </resources>
 <connections/>
</ui>
